# file: C:\Users\rcasa\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\jinja2\async_utils.py
# hypothesis_version: 6.135.17

['V', '__aiter__', '__module__', '__name__', '__qualname__', 't.AsyncIterator[V]', 't.Awaitable[V]', 't.Iterator[V]', 'te.Self']